content/app-dev/create-provider/new-interface.md

```diff
@@ -81,8 +81,13 @@ wash new interface payments
 ```
 
 This command creates a new interface project in the subdirectory 'payments'. 
-Wasmcloud interfaces are defined in the [smithy](https://awslabs.github.io/smithy/) IDL. 
-When changes are made to the `.smithy` file, a code generator generates code for the selected target languages - currently only Rust is supported, and more languages will be supported in the future. The `make` command runs the code generator to generate the rust library, and compiles it. The generated library defines function signatures and data structures that will be shared by the Payments capability provider and the actor that calls the provider.
+Wasmcloud interfaces are defined in the [smithy](https://awslabs.github.io/smithy/) IDL. When changes are made to the `.smithy` file, a code generator generates code for the selected target languages - currently only Rust is supported, and more languages will be supported in the future. 
+
+The `wash new interface` command will ask for some information in order to fill in some meta-data in the generated code. First it will ask if you want to generate a `converter-interface` or a `factorial-interface`. The former is for an interface between actors so you would want to choose `factorial-interface` which is between actor and a capability provider. Use arrow-keys to select the right one.
```

Comment:
Instead of `Use arrow-keys to select the right one.`, can we say:
```
Use UP/DOWN arrow-keys or the j/k keys to select the right one.
```

This is a suggestion as some keyboards don't have arrow keys, and the vim-style J/K navigation works too.